### PR TITLE
Story1

### PR DESCRIPTION
**Pull request by JamesMcGraaw**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,20 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+<head>
+    <meta charset="utf-8"/>
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico"/>
```
Comment (nealioa) [inline public/index.html]: This file doesn't appear to exist. What's the point in referencing it?

Same comment applies to logo192.png and manifest.json below.

File: src/components/Header/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                            <a className="nav-link" href="#">Full time</a>
+                        </li>
+                        <li className="nav-item">
+                            <a className="nav-link" href="#">Part time</a>
+                        </li>
+                        <li className="nav-item">
+                            <a className="nav-link">Contract</a>
+                        </li>
+                    </ul>
+                            <button className="btn btn-outline-success" type="submit">Post a job</button>
```
Comment (nealioa) [inline src/components/Header/index.js]: This line seems to be indented too far.

File: src/components/RecentJobs/index.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                        <td className="row-span-2">{roundedSalary ? `£${roundedSalary}` : ""}</td>
+                        <td className="row-span-2">
+                            {job.skills.map((skill, index) => {
+                                return <button key={index} className="btn btn-secondary">{skill.skill}</button>
+                            })}
+                        </td>
+                    </tr>
+                );
+            });
+    }
```
Comment (nealioa) [inline src/components/RecentJobs/index.js]: This bracket should be indented.

File: src/components/RecentJobs/index.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            <table className="table table-dark table-striped">
+                <thead>
+                </thead>
+                <tbody></tbody>
+            </table>
+        </div>
+    );
+};
+
+export default RecentJobs;
```
Comment (nealioa) [inline src/components/RecentJobs/index.js]: To be consistent with other component files, the semicolon should be deleted (or semicolons added to the others).

**Review by nealioa — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (daedalus-9) [reply public/index.html]: Amended

File: src/components/Header/index.js (hunk 2 above)
Comment (daedalus-9) [reply src/components/Header/index.js]: Amended

File: src/components/RecentJobs/index.js (hunk 3 above)
Comment (daedalus-9) [reply src/components/RecentJobs/index.js]: Amended

File: src/components/RecentJobs/index.js (hunk 4 above)
Comment (daedalus-9) [reply src/components/RecentJobs/index.js]: Amended

**Review by nealioa — APPROVED** [review]

**Pull request merged**